webapp/pages/post/_id/_slug/index.vue

```diff
@@ -89,7 +89,19 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !this.blocked"
```

Comment:
```suggestion
          v-if="showNewCommentForm && !post.author.isBlocked"
```

I think that's enough and you can remove `blockedByPostAuthor`.